src/environment.rs

```diff
@@ -151,6 +156,53 @@ impl Environment {
     pub unsafe fn close_db(&mut self, db: Database) {
         ffi::mdb_dbi_close(self.env, db.dbi());
     }
+
+    /// Get database statistics
+    pub fn stat(&self) -> Result<Stat> {
+        unsafe {
+            let mut stat = Stat(mem::uninitialized());
```

Comment:
I think this should be `mem::zeroed()`, to match the common idiom in C.  I think there's a chance that as written this is UB since a few lines below you are taking a reference to it.